.github/ISSUE_TEMPLATE.md

```diff
@@ -1,15 +1,31 @@
-<!-- bug reporting without issue template will be closed automatically -->
+<!-- Following this template is mandatory to open an issue. Please use English -->
 
-## Expected behavior, english is required
+## Describe the behavior that you expect from SpaceVim
 
-## The reproduce ways from Vim starting (Required!)
+**[This is a mandatory section]**
 
-## Debug info
+If possible, in addition to the description, please also provide a link to the relevant SpaceVim documentation that describes the expected behaviour/feature.
 
-Please press <kbd>SPC h I</kbd>, debug info will be put into clipboard, then paste all content below.
+## Describe the problem
 
-## Screenshots
+**[This is a mandatory section]**
 
-If you have any screenshots for this issue, please upload here. BTW you can use https://asciinema.org/ for recording video in terminal.
+Please describe the observed problem in simple and clear terms.
 
-<!-- please remove the issue template when request for a feature -->
+## Describe steps to reproduce the problem
+
+**[This is a mandatory section]**
+
+Please list all the steps from the point the Vim session was started until the problem was observed, in simple and clear terms.
+
+## Provide debug information from SpaceVim
+
+**[This is a mandatory section]**
+
+From within SpaceVim, please press `SPC h I` (ie: press and release`space`, followed by pressing and releasing the `h` key, followed by pressing and releasing the `I` key (capital I)). This will make SpaceVim print out debug information and also copy it to the clipboard. Please paste this debug information below in entirety.
```

Comment:
it should be "press ~~and release~~ `space`, "

Reply:
On my system, I have to 'tap' `space` (so press and release, rather than press and hold), then tap `h` and finally tap `I`. I assume that is the expected way to use these keys ?

Basically, I wanted to prevent `SPC h I` being interpreted as:

- Press space and keep it pressed, then
- press h and keep it pressed, then
- press I

Does that make sense or have I misunderstood something ?

Reply:
Have you tried this sequence (press together <kbd>Space</kbd>+<kbd>h</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd>, than release all keys) ?

On my GNU/Linux laptop, it opens `issue_report.md` in a new tab (so it basically does the same thing... an emacs way :see_no_evil:)

Reply:
@Rom1deTroyes: Thanks fore the response.  I see that too!

FWIW: I tried spacemacs recently and even it uses the 'tap' dynamic. So whenever they say `SPC a b` they mean taps.

I agree with you - the question here is how best to disambiguate 'press'. Maybe I should just change the text to:

```From within SpaceVim, please tap the following in order: `SPC h I` (ie: tap `space`, then tap `h` and then tap `I` (uppercase `i`). This....```

Do you think that's clearer ?

Reply:
yeah, LGTM

Reply:
Thanks @wsdjeg : I'll submit a new PR.

Reply:
Ok, but why not just push to this pr?